### ISSUE TITLE
8 space hanging-indent incorrectly marked as incorrect

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#153]
Originally reported by: **BitBucket: [thapar](http://bitbucket.org/thapar), GitHub: @thapar?**

---

PEP8 (http://legacy.python.org/dev/peps/pep-0008/#indentation) states "further indentation should be used to clearly distinguish itself as a continuation line." And in the subsequent example:

```
# More indentation included to distinguish this from the rest.
def long_function_name(
        var_one, var_two, var_three,
        var_four):
    print(var_one)
```

However, with pylint, if 8 spaces are used for a hanging indent, it throws an error. If 4 spaces are used, no error. Here is the problem code with 8 spaces:

```
password = ''.join(random.choice(
        string.ascii_lowercase + string.digits) for x in range(8)
)
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/153
